ts/components/select.ts

```diff
@@ -168,7 +168,11 @@ export default (options: InitOptions): Select => ({
 
     return textarea.value
   },
-  getFocusables () { return [...this.$el.querySelectorAll('li, input')] },
+  getFocusables () {
+    const focusables = this.$el.querySelectorAll('li, input')
+
+    return focusables.length > 0 ? [...focusables] : [...this.$root.querySelectorAll('li, input')]
```

Comment:
Follow the @putera suggestion with a little change

```suggestion
    const focusables = this.$el?.querySelectorAll('li, input') ?? []

    return focusables.length > 0 ? [...focusables] : [...this.$root?.querySelectorAll('li, input')]
```